src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -51,6 +53,22 @@
 		)
 	].sort(sortIcTokens);
 
+	let allTokens: Token[] = [];
+	$: allTokens = [
```

Comment:
Can't we use the global store `$tokens` to get all the tokens?
Also shouldn't we filter test tokens?

Reply:
I used `networkTokens` since it will keep the same behavior if we select ICP network: if it's Chain Fusion, `$tokens` is the same as `$networkTokens` but without the test tokens (Sepolia for example).

However, you are right that the ICP test tokens appear there, I wanted to do a separate PR to remove them.

Do you prefer here?

Reply:
Is that the expected behavior from Artem and Mary? I actually would expect that when I access "Manage Tokens" I can always manage all my tokens regardless of the selected network. 🤷‍♂️

Reply:
Note that my comment above about testnet was incorrect I think. For ERC20 I can also set custom ERC20 on Sepolia.

Reply:
> Is that the expected behavior from Artem and Mary? I actually would expect that when I access "Manage Tokens" I can always manage all my tokens regardless of the selected network. 🤷‍♂️

Ah, that I didn't know, I thought the user should manage only the tokens in the selected network. Let me ask them

Reply:
I wrote to Artem and he said

Ok, it looks like it indeed m

> akes sense to show only the tokens related to the selected network. Is it possible?
> Maybe some small hint under the search would help with the context?
> e.g.
> Managing token for Internet Computer
> Managing token for ChainFusion

he made edits in figma

https://www.figma.com/design/2Wi7w1kenBrOhKgKo4iD3Z/OISY-Layouts?node-id=1354-5273&t=JO5QzcGQyyYI2EhX-0

Reply:
Thanks for double checking. I also quickly discussed with him to be sure I understand everything. Let's do that then.